src/app/core/services/config/config.service.ts

```diff
@@ -130,6 +140,22 @@ export class ConfigService {
     })
   }
 
+  checkForAppUpdates() {
+    const playstoreURL = DefaultGooglePlaystoreAppURL + DefaultPackageName
+    return Promise.all([
+      this.http
+        .get(playstoreURL, { responseType: 'text' })
+        .toPromise()
+        .then(res => parseVersion(res))
+        .catch(e => DefaultAppVersion),
+      this.appConfig.getAppVersion()
+    ]).then(([playstoreVersion, currentVersion]) => {
+      if (ver.gt(ver.coerce(playstoreVersion), ver.coerce(currentVersion)))
```

Comment:
I am not sure if we should coerce here. Sometimes we release hotfixes as they are urgent and naming for them is like 0.4.3.1 (not in semantic spec) but coercing it will change it to 0.4.3 and the app will not prompt for update.

Reply:
Ohh okay. I added this for of the `-alpha` version, so should we only do for production releases?

Reply:
Yes because we can't for alpha releases even if we want to. Alpha releases are only available for people on the invited lists when they are logged into their account. Publicly, it will always show the production release.

Reply:
Ok will make the changes.